src/ParallelDataCollector.cpp

```diff
@@ -956,6 +970,45 @@ namespace splash
         dataset.close();
     }
 
+    CollectionType* ParallelDataCollector::readDataSetMeta(H5Handle h5File,
+            int32_t id,
+            const char* name,
+            const Dimensions dstBuffer,
+            const Dimensions dstOffset,
+            const Dimensions srcOffset,
+            Dimensions &sizeRead,
+            uint32_t& srcDims)
+    throw (DCException)
+    {
+        log_msg(2, "readDataSetMeta");
+
```

Comment:
You should go ahead and document all function behavior using ascii art flow
charts... like a pro
On Nov 10, 2015 8:22 AM, "Axel Huebl" <notifications@github.com> wrote:

> In src/ParallelDataCollector.cpp
> <https://github.com/ComputationalRadiationPhysics/libSplash/pull/203#discussion_r44426550>
> :
>
> > @@ -956,6 +970,45 @@ namespace splash
> >          dataset.close();
> >      }
> >
> > +    CollectionType* ParallelDataCollector::readDataSetMeta(H5Handle h5File,
> > +            int32_t id,
> > +            const char* name,
> > +            const Dimensions dstBuffer,
> > +            const Dimensions dstOffset,
> > +            const Dimensions srcOffset,
> > +            Dimensions &sizeRead,
> > +            uint32_t& srcDims)
> > +    throw (DCException)
> > +    {
> > +        log_msg(2, "readDataSetMeta");
> > +
>
> probably it would be easier to read in our venerable age if we would add
> more ascii art to the blocks. kids these day...!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ComputationalRadiationPhysics/libSplash/pull/203/files#r44426550>
> .
>